### ISSUE TITLE
Build is broken

### ISSUE THREAD

**Issue by fschoell** [pinax-network/substreams-antelope-tokens#7]
@YaroShkvorets 

```
➜  substreams-antelope-tokens git:(main) ✗ make build
cargo build --target wasm32-unknown-unknown --release
   Compiling antelope_tokens v0.3.7 (/Users/work/RustRover/substreams-antelope-tokens)
error[E0599]: no method named `transaction_traces` found for struct `Block` in the current scope
  --> src/maps.rs:13:23
   |
13 |     let items = block.transaction_traces().flat_map(|trx| {
   |                       ^^^^^^^^^^^^^^^^^^ help: there is a method with a similar name: `all_transaction_traces`

error[E0599]: no method named `transaction_traces` found for struct `Block` in the current scope
  --> src/maps.rs:78:23
   |
78 |     let items = block.transaction_traces().flat_map(|trx| {
   |                       ^^^^^^^^^^^^^^^^^^ help: there is a method with a similar name: `all_transaction_traces`

error[E0308]: mismatched types
   --> src/maps.rs:156:74
    |
156 |     let items = block.actions::<abi::actions::Transfer>(&[]).filter_map(|(action, action_trace, trx)| {
    |                                                                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |                                                                          |
    |                                                                          expected a tuple with 2 elements, found one with 3 elements
    |                                                                          expected due to this
    |
    = note: expected tuple `(Transfer, &ActionTrace)`
               found tuple `(_, _, _)`

Some errors have detailed explanations: E0308, E0599.
For more information about an error, try `rustc --explain E0308`.
error: could not compile `antelope_tokens` (lib) due to 3 previous errors
make: *** [build] Error 101
```

**Comment by YaroShkvorets** [comment]
Found the issue. Cargo dependency resolver is pulling `substreams-antelope v0.3.4` (as I expected) but `substreams-antelope` is pulling `substreams-antelope-core v0.4.1` because it has a local dependnecy `substreams-antelope-core = { version = "0.*", path = "./core" }`. Good to know.
Let me release new version.

**Comment by YaroShkvorets** [comment]
fixed in 0.3.8
